src/main/java/org/commcare/core/process/CommCareInstanceInitializer.java

```diff
@@ -138,7 +139,15 @@ protected TreeElement loadFixtureRoot(ExternalDataInstance instance,
 
             TreeElement root = fixtureBases.retrieve(key);
             if (root == null) {
-                FormInstance fixture = SandboxUtils.loadFixture(mSandbox, refId, userId);
+                IStorageUtilityIndexed<FormInstance> fixtureStorage = null;
+                if (mPlatform != null) {
+                    fixtureStorage = mPlatform.getFixtureStorage();
```

Comment:
If I'm understanding this change properly, this means that if there are app fixtures in both user storage and app storage, we default to the ones in app storage. Is that intended? If so that's fine, I just don't know which is correct or if there is a clear correct answer for all cases.

Reply:
Updated to attempt both (falling back to user storage if it isn't in app storage)